CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 3.1)
+cmake_minimum_required (VERSION 2.8)
```

Comment:
You can't just do that, at least `CMAKE_CXX_STANDARD` doesn't exist in cmake 2.8. Will try to finish the patch but it's more involved than that.